docs/getting-started-guides/docker.md

```diff
@@ -0,0 +1,71 @@
+## Running kubernetes locally via Docker
```

Comment:
I don't think that's necessary if you use $(boot2docker ip) instead of local host.